js/agency.js

```diff
@@ -5,15 +5,17 @@
  */
 
 // jQuery for page scrolling feature - requires jQuery Easing plugin
-$(function() {
-    $('a.page-scroll').bind('click', function(event) {
-        var $anchor = $(this);
-        $('html, body').stop().animate({
-            scrollTop: $($anchor.attr('href')).offset().top
-        }, 1500, 'easeInOutExpo');
-        event.preventDefault();
-    });
-});
+
+// Handling smooth scrolling in CSS. But leaving this here we need to revert due to browser support issues
```

Comment:
I suggest removing anything you no longer need at the moment (vs commenting it out). It will still be available on git in the off-chance you want it back.

Reply:
Thanks for the feedback, @carloslima. I've removed it